pandas/core/generic.py

```diff
@@ -4,7 +4,7 @@
 import collections
 import datetime as dt
 import gc
-import json
+import json as js
```

Comment:
this is non-standard and not obvious - does it work to just import the functions from json, so `from json import loads, jumps` and then use those directly?

Reply:
I have changed it based on your request